src/output.rs

```diff
@@ -251,3 +251,42 @@ impl Output for Status {
         Ok(Status(result?.exit_status))
     }
 }
+
+/// Using [`bool`] as the return type for [`cmd!`] will return `true` if
+/// the command returned successfully, and `false` otherwise:
+///
+/// ```
+/// use cradle::*;
+///
+/// if !cmd!(%"echo foo") {
+///     panic!("echo should not fail!");
+/// }
```

Comment:
```suggestion
/// if !cmd!(%"which cargo") {
///     panic!("`cargo` is not installed");
/// }
```

Maybe this is better?

Reply:
Agreed, I was actually thinking that `assert!`-ing a bool isn't super clear.